src/dotnet/APIView/APIViewWeb/LeanControllers/APIRevisionsController.cs

```diff
@@ -111,5 +109,19 @@ public async Task<ActionResult<APIRevisionListItemModel>> ToggleReviewApprovalAs
             }
             return new LeanJsonResult(apiRevision, StatusCodes.Status200OK);
         }
+
+        /// <summary>
+        /// Endpoint used by Client SPA for getting associated Pull Request
+        /// </summary>
+        /// <param name="reviewId"></param>
+        /// <param name="revisionId"></param>
+        /// <returns></returns>
+        /// 
+        [HttpGet("{reviewId}/{revisionId}/associatedPullRequest", Name = "GetAssociatedPullRequest")]
```

Comment:
Please can you use `apiRevisionId`

Reply:
Yes, I remember you saying that now. Thanks